libs/designer/src/lib/ui/panel/panelroot.tsx

```diff
@@ -192,6 +197,19 @@ export const PanelRoot = (props: PanelRootProps): JSX.Element => {
     dispatch(expandPanel());
   }, [dispatch]);
 
+  const [showDeleteModal, setShowDeleteModal] = useState(false);
+  const handleDeleteClick = () => setShowDeleteModal(true);
+  const handleDelete = () => {
+    // TODO: 12798935 Analytics (event logging)
+    if (isScopeOperation(operationInfo?.type)) {
```

Comment:
For subgraphnodes i do see the delete method calling deleteSwitchCase as well, so if you explicitly delete case maybe not all graphs get deleted. @rllyy97 to confirm on this.

Reply:
I'll take a look to see if I can add handling for cases under a switch case.

Reply:
Yes Priti is correct, you'll call the `deleteGraphNode` function with the subgraph/case id and node object, then `deleteSwitchCase` with the subgraph/case id and the containing graph node id (the actual switch id)

Reply:
Fixed

Reply:
![nestedGraphDeletionBugFixFinal](https://github.com/Azure/LogicAppsUX/assets/72991696/fedf5d95-92b3-4cd4-b0e1-9e542c7fcffd)

Reply:
Ignore bad quality of the GIF - had to make some edits to reduce size. There's no weird residue of the delete modal after accepting/declining the prompt lol